integration/kubernetes/init.sh

```diff
@@ -108,12 +108,12 @@ trap 'sudo -E sh -c "rm -r "${kubeadm_config_file}""' EXIT
 
 if [ "${BAREMETAL}" == true ] && [[ $(wc -l /proc/swaps | awk '{print $1}') -gt 1 ]]; then
```

Comment:
We need let the  "BAREMETAL" check separate from the swap check. So what about:
"
if [[ $(wc -l /proc/swaps | awk '{print $1}') -gt 1 ]]; then
         sudo swapoff -a || true
fi

if [ "${BAREMETAL}" == true &&  "$(command -v kubelet)" != "" ]; then
       #reinstall kubelet to do deep cleanup
      	info "reinstall kubeadm, kubelet before initialize k8s"
	bash -f "${SCRIPT_PATH}/../../.ci/install_kubernetes.sh"	
fi
"

Reply:
The Arm Ci fails as the "wc -l /proc/swaps | awk '{print $1}'" is "1" in the Arm server which results in the "reinstall k8s" ops didn't start.

Reply:
I'm not sure about this, @jongwu.

The baremetal check to disable swap has always been there and shouldn't cause an issue (it wasn't already).
I'll split the check, and let it run a few more times.

Reply:
Ah, now I see, sorry, my bad!